src/jit/codegencommon.cpp

```diff
@@ -10265,7 +10265,52 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 /*****************************************************************************
  *
- *  Generates code for an EH funclet prolog.
+ *  Generates code for an EH funclet prolog. WIP.
+ *
+ *  Funclets have the following incoming arguments:
+ *
+ *      catch/filter-handler: ebp+8 = CallerSP, ebp+12 = the exception object that was caught (see GT_CATCH_ARG)
```

Comment:
This calling convention for funclets looks odd. Was it inspired by something?

I think the natural calling convention for funclets would be:

```
ecx = established frame
edx = exception object
ebp, esi, edi, ebx = restored to what they were when the control flow left the method last time
```